src/js/msp/MSPHelper.js

```diff
@@ -363,7 +363,8 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 for (var i = 0, needle = 0; i < (data.byteLength / 3); i++, needle += 3) {
                     // main for loop selecting the pid section
                     for (var j = 0; j < 3; j++) {
-                        PIDs[i][j] = data.readU8();
+                        MSP_PIDS[i][j] = data.readU8();
```

Comment:
Looking at this again, I think the current naming will be confusing - I think `PIDS_ORIGINAL` and `ADVANCED_TUNING_ORIGINAL` or similar will make it easier for others to realise that these are backing values and not to be modified.

Reply:
I agree that `MSP_PIDS` is a bit of a misleading name, but I'm not convinced that `PIDS_ORIGINAL` is any better as to me it implies that it is a default rather than the value that's currently in use. I don't think that "current" would be any better either.

I think the best disambiguation would be to rename both `PIDs` to `PIDS_WORKING` and `MSP_PIDS` to `PIDS_ACTIVE` - Working being the values that are being modified within the configurator and active being the values that are currently loaded within BetaFlight. For this to make the most sense, the preexisting `PIDs` variable would need to be renamed which would make it inconsistent with the key in the backup file's profiles section.

Reply:
Good point, `_ORIGINAL` isn't great either. How about `PIDs` (to retain compatibility with the backup) and `PIDS_ACTIVE`?

Reply:
I think that's about as good as it can be without modifying the existing variables, I'll change them to `*_ACTIVE`.